packages/cli/src/cli/commands/dev.ts

```diff
@@ -116,6 +132,16 @@ export default defineCommand({
           ...(args.preset ? ["--preset", args.preset] : []),
         ];
 
+        if (selectedIndexers.length === 0) {
+          for (const indexer of apibara.indexers) {
+            apibara.logger.info(`Indexer ${blueBright(indexer.name)} started`);
+          }
+        } else {
+          for (const indexer of selectedIndexers) {
+            apibara.logger.info(`Indexer ${blueBright(indexer)} started`);
+          }
+        }
+
```

Comment:
This should be in `runtime/dev.ts` using the indexer's logger. You need to change `createIndexer` to return `{ indexer, logger }`.

It should print this right before `runWithReconnect`.